specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/wsn/client.should.disconnect.if.wrong.control.bytes.value/request.rpt

```diff
@@ -0,0 +1,30 @@
+connect tcp://127.0.0.1:8001
```

Comment:
This test is not valid, please remove it. The reason is the SPEC.md does not specify what the control bytes are, and they can be anything the implementation chooses. There is nothing invalid about them being 01010100 instead of 01010102.